pkg/reconciler/shared/tektonconfig/upgrade/pre_upgrade_test.go

```diff
@@ -82,7 +82,7 @@ func TestUpgradePipelineProperties(t *testing.T) {
 		Spec: v1alpha1.TektonConfigSpec{
 			Pipeline: v1alpha1.Pipeline{
 				PipelineProperties: v1alpha1.PipelineProperties{
-					EnableStepActions: ptr.Bool(false),
```

Comment:
Any reason for changing this ? Shouldn't we instead add another test ?

Reply:
I just didn't want to add another test case. 😆

But if the logic of `SetDefaults` changes in the future, for example, changing the default value from `false` to `true`, it may cause the unit test to fail.
I will add another case.

Reply:
I have resubmitted the code and made adjustments to the unit test.
Please help review it when you have time. Thank you!